Documentation/network/bgp-control-plane.rst

```diff
@@ -17,292 +17,509 @@ reachability within the cluster.
+When the load balancer class is set to ``io.cilium/bgp-control-plane`` or unspecified,
+Cilium will announce the ingress IPs of the Service. Otherwise, Cilium will not announce
+the ingress IPs of the Service.
 
-When running in ``IPv6`` single stack or when the administrator needs to
-manually define the instantiated BGP server's router ID a Kubernetes
-annotation can be placed on the node.
+externalTrafficPolicy
+~~~~~~~~~~~~~~~~~~~~~
 
-The annotation takes the following syntax:
+When the Service has ``externalTrafficPolicy: Cluster``, BGP Control Plane
+unconditionally advertises the ingress IPs of the selected Service. When the
+Service has ``externalTrafficPolicy: Local``, BGP Control Plane keeps track of
+the endpoints for the service on the local node and stops advertisement when
+there's no local endpoint.
 
-::
+Validating Advertised Routes
+----------------------------
 
-   cilium.io/bgp-virtual-router.{asn}="router-id=127.0.0.1"
+Get all IPv4 unicast routes available:
 
-The above annotation syntax should replace ``{asn}`` with the local ASN
-of the ``CiliumBGPVirtualRouter`` you are setting the provided router ID
-for.
+.. code-block:: shell-session
 
-When the ``BGPControlPlane`` evaluates a ``CiliumBGPPeeringPolicy`` with
-a ``CiliumBGPVirtualRouter`` it also searches for an annotation which
-targets the aforementioned ``CiliumBGPVirtualRouter`` local ASN.
+   $ cilium bgp routes available ipv4 unicast
+   Node                              VRouter   Prefix        NextHop   Age    Attrs
+   node0                             64512     10.1.0.0/24   0.0.0.0   17m42s [{Origin: i} {Nexthop: 0.0.0.0}]
 
-If found it will use the provided router ID and not attempt to use the
-IPv4 address assigned to the node.
+Get all IPv4 unicast routes available for a specific vrouter:
 
-Local Listening Port
-^^^^^^^^^^^^^^^^^^^^
+.. code-block:: shell-session
 
-By default the ``GoBGP BGPRouterManager`` will instantiate each virtual
-router without a listening port.
+   $ cilium bgp routes available ipv4 unicast vrouter 64512
+   Node                              VRouter   Prefix        NextHop   Age    Attrs
+   node0                             64512     10.1.0.0/24   0.0.0.0   17m42s [{Origin: i} {Nexthop: 0.0.0.0}]
 
-It is possible to deploy a virtual router which creates a local
-listening port where BGP connections may take place.
+Get IPv4 unicast routes advertised to a specific peer:
 
-If this is desired the following annotation can be provided
+.. code-block:: shell-session
 
-::
+   $ cilium bgp routes advertised ipv4 unicast peer 10.0.0.1
+   Node                              VRouter   Prefix        NextHop   Age    Attrs
+   node0                             64512     10.1.0.0/24   10.0.0.2  17m42s [{Origin: i} {AsPath: } {Nexthop: 10.0.0.2} {LocalPref: 100}]
 
-   cilium.io/bgp-virtual-router.{asn}="local-port=45450"
 
-Neighbors
-^^^^^^^^^
+Neighbor Options
```

Comment:
Can you also add `families` section under neighbor options.

Reply:
Let me handle this in the separate PR 🙏